### DISCUSSION TITLE
Gimmick - Using the GimkitCheat on Chromebooks

### DISCUSSION THREAD

**Discussion by VillainsRule** [TheLazySquid/GimkitCheat#75] [Show and tell]
I've built [Gimmick](https://github.com/VillainsRule/Gimmick), a website that automatically embeds the latest version of your GimkitCheat in a copy of Gimkit. It currently only allows joining (Google auth issues), but I plan to work on expanding it more.

I'm always looking for help & feedback, so lmk what you think!

**Comment by TheLazySquid** [comment]
Interesting! Right now there isn't any good way to use this on managed computers, so with a little hosting this could be very useful.

**Reply by helloIlovegimkit** [reply]
Yes, on windows as microsoft forces you to install microsoft edge and for me google chrome is the only one that's managed (for me).


(if that makes sense)

**Reply by VillainsRule** [reply]
I'm sadly unable to permanently host this because I just don't have the equipment - although it could. I have over five domains currently unused, so if anyone has a server, we could work together.

**Reply by helloIlovegimkit** [reply]
I'm able to host it for a while as I have a raspberry pi

(for specifically how long you need me to)

**Reply by helloIlovegimkit** [reply]
and also it would be harder for schools to block as they'd have to trace my raspberry pi and even then I can change the top level domain

**Reply by VillainsRule** [reply]
> * to Cloudflare tunnels, almost impossible to block wo/ taking down half the internet for the students

I currently host it permanently on my domain. Following that comment, I was able to gain access to a complete desktop setup.

https://gim.villainsrule.xyz

**Reply by VillainsRule** [reply]
Furthermore, you fundamentally fail to understand how domains work. Please do some research.

(Cloudflare is also used by only 20% of the web and less than 5% of school sites because they have their own infra...NO idea where you got half from)

**Comment by Zaickerzn** [comment]
This might work using GitHub code spaces if you made do that

**Reply by VillainsRule** [reply]
I can confirm it works on Github Codespaces as well as CodeSandbox.
It does not work on StackBlitz.

**Comment by MRHACKZ001** [comment]
do you guys know ow to get the day one gimkit skin

**Reply by VillainsRule** [reply]
Probably buy an account, realistically.

**Reply by TheLazySquid** [reply]
This isn't the place to discuss this, but you can't obtain it unless you have an account that was there to buy it.
